### PR TITLE
cmd/geth: added copyright and license information

### PR DESCRIPTION
**Pull request by obscuren**
As per GNU GPL requirement I've added the copyright and the license
information as a subcommand as well as a copyright notice when
displaying the help command.

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Sun Aug 21 21:52:48 UTC 2016_

File: cmd/geth/usage.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ import (
 var AppHelpTemplate = `NAME:
    {{.App.Name}} - {{.App.Usage}}
 
+   Copyright 2013-2015 The go-ethereum Authors
```
Comment (karalabe) [inline cmd/geth/usage.go]: 2016? :P

Comment (obscuren) [inline cmd/geth/usage.go]: I was only testing you guys!

Comment (karalabe) [inline cmd/geth/usage.go]: ;)

**Comment by karalabe** [comment]
LGTM :+1:

**Comment by Arachnid** [comment]
:+1:

**Pull request merged**